roles/directory/files/nginx.conf

```diff
@@ -0,0 +1,6 @@
+server {
+  location / {
+    add_header 'Access-Control-Allow-Origin' "*" always;
+    proxy_pass https://spaceapi.fixme.ch/directory.json
+  }
+}
```

Comment:
There are some files like this with a missing line ending at the end. I'm not sure if this may cause problems.

Reply:
shouldn't, but i can fix that afterwards so the commits on the same files look better in the future